docs/integrations/microsoft-azure/sql.md

```diff
@@ -75,51 +78,180 @@ For details on Azure SQL logs and metrics, see [Enable logging](https://docs.mic
+   - `location`. The region to which the resource name belongs to.
+   - `subscription_id`. Id associated with a subscription where resource is present.
+   - `resource_group`. The resource group name where the Azure resource is present.
+   - `provider_name`. Azure resource provider name (for  ex Microsoft.Storage).
+   - `resource_type`. Azure resource type (for ex storageaccounts).
+   - `resource_name`. The name of the resource (for ex Azure SQL Server name).
+   - `service_type`. Type of the service that can be accessed from with a azure resource.
+   - `service_name`. Services that can be accessed from within a azure resource (Ex Azure SQL databases in Azure SQL Server).
+
+3. Create the fields if it is not present. Refer to [create and manage fields](/docs/manage/fields/#manage-fields).
+
+### Configure Field Extraction Rules
+
+Create a Field Extraction Rule (FER) by following the instructions [here](/docs/manage/field-extractions/create-field-extraction-rule/). If the FER already exists with same name, then skip this step.
+
+* **Azure Location Extraction FER**
+
+   ```sql
+   Rule Name: AzureLocationExtractionFER
+   Applied at: Ingest Time
+   Scope (Specific Data): tenant_name=*
+   ```
+
+   ```sql title="Parse Expression"
+   json "location", "properties.resourceLocation", "properties.region" as location, resourceLocation, service_region nodrop
+   | replace(toLowerCase(resourceLocation), " ", "") as resourceLocation
+   | if (!isBlank(resourceLocation), resourceLocation, location) as location
+   | if (!isBlank(service_region), service_region, location) as location
+   | if (isBlank(location), "global", location) as location
+   | fields location
+   ```
+
+* **Resource ID Extraction FER**
+
+   ```sql
+   Rule Name: AzureResourceIdExtractionFER
+   Applied at: Ingest Time
+   Scope (Specific Data): tenant_name=*
+   ```
+
+   ```sql title="Parse Expression"
+   json "resourceId"
+   | toUpperCase(resourceId) as resourceId
+   | parse regex field=resourceId "/SUBSCRIPTIONS/(?<subscription_id>[^/]+)" nodrop
+   | parse field=resourceId "/RESOURCEGROUPS/*/" as resource_group nodrop
+   | parse regex field=resourceId "/PROVIDERS/(?<provider_name>[^/]+)" nodrop
+   | parse regex field=resourceId "/PROVIDERS/[^/]+(?:/LOCATIONS/[^/]+)?/(?<resource_type>[^/]+)/(?<resource_name>.+)" nodrop
+   | parse regex field=resource_name "(?<parent_resource_name>[^/]+)(?:/PROVIDERS/[^/]+)?/(?<service_type>[^/]+)/?(?<service_name>.+)" nodrop
+   | if (isBlank(parent_resource_name), resource_name, parent_resource_name) as resource_name
+   | fields subscription_id, location, provider_name, resource_group, resource_type, resource_name, service_type,service_name
+   ```
+### Configure metric rules
+
+  * **Azure Observability Metadata Extraction Metric Rule Service Level**
+
+      In case this rule is already exists then no need to create again.
+
+      ```sql
+      Rule Name: AzureObservabilityMetadataExtractionServiceLevel
```

Comment:
Should this be `AzureObservabilityMetadataExtractionMetricRuleServiceLevel`?

Reply:
actually it was making the Rule name very big, unfortunately we do not have any shortform for metric rules similar to FER for Field Extraction Rules.

Reply:
Oh I see. Yes that would be reeeeally long haha.